Handle quotation marks

```
Error resolving with path organization?name=National+Centre+of+Scientific+Research+%22Demokritos%22: Entity resolver returned 502 for organization type and {"name"=>"National Centre of Scientific Research \"Demokritos\""} params.
```

```
entity-resolver_1         | 2018/11/05 12:28:33 [SPARQL] SELECT ?id
entity-resolver_1         | 			WHERE {
entity-resolver_1         | 				?id a <http://xmlns.com/foaf/0.1/Organization> .
entity-resolver_1         | 				?id <http://www.w3.org/2004/02/skos/core#prefLabel> "National Centre of Scientific Research "Demokritos"" .
entity-resolver_1         | 			} returned error: Query: SPARQL request failed: 400 Bad Request. Response body: 
entity-resolver_1         | SPARQL-QUERY: queryStr=SELECT ?id
entity-resolver_1         | 			WHERE {
entity-resolver_1         | 				?id a <http://xmlns.com/foaf/0.1/Organization> .
entity-resolver_1         | 				?id <http://www.w3.org/2004/02/skos/core#prefLabel> "National Centre of Scientific Research "Demokritos"" .
entity-resolver_1         | 			}
entity-resolver_1         | java.util.concurrent.ExecutionException: org.openrdf.query.MalformedQueryException: Lexical error at line 4, column 108.  Encountered: "\"" (34), after : "Demokritos"
entity-resolver_1         | 	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
entity-resolver_1         | 	at java.util.concurrent.FutureTask.get(FutureTask.java:192)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.BigdataServlet.submitApiTask(BigdataServlet.java:281)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.QueryServlet.doSparqlQuery(QueryServlet.java:653)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.QueryServlet.doPost(QueryServlet.java:273)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.RESTServlet.doPost(RESTServlet.java:269)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.MultiTenancyServlet.doPost(MultiTenancyServlet.java:193)
entity-resolver_1         | 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
entity-resolver_1         | 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
entity-resolver_1         | 	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:769)
entity-resolver_1         | 	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:585)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
entity-resolver_1         | 	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:577)
entity-resolver_1         | 	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:223)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1125)
entity-resolver_1         | 	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:515)
entity-resolver_1         | 	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:185)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1059)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:215)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:110)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)
entity-resolver_1         | 	at org.eclipse.jetty.server.Server.handle(Server.java:497)
entity-resolver_1         | 	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:311)
entity-resolver_1         | 	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:248)
entity-resolver_1         | 	at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:540)
entity-resolver_1         | 	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:610)
entity-resolver_1         | 	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:539)
entity-resolver_1         | 	at java.lang.Thread.run(Thread.java:748)
entity-resolver_1         | Caused by: org.openrdf.query.MalformedQueryException: Lexical error at line 4, column 108.  Encountered: "\"" (34), after : "Demokritos"
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.Bigdata2ASTSPARQLParser.parseQuery2(Bigdata2ASTSPARQLParser.java:404)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.QueryServlet$SparqlQueryTask.call(QueryServlet.java:716)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.QueryServlet$SparqlQueryTask.call(QueryServlet.java:670)
entity-resolver_1         | 	at com.bigdata.rdf.task.ApiTaskForIndexManager.call(ApiTaskForIndexManager.java:68)
entity-resolver_1         | 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
entity-resolver_1         | 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
entity-resolver_1         | 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
entity-resolver_1         | 	... 1 more
entity-resolver_1         | Caused by: com.bigdata.rdf.sail.sparql.ast.TokenMgrError: Lexical error at line 4, column 108.  Encountered: "\"" (34), after : "Demokritos"
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilderTokenManager.getNextToken(SyntaxTreeBuilderTokenManager.java:3994)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.jj_ntk(SyntaxTreeBuilder.java:9633)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.RDFLiteral(SyntaxTreeBuilder.java:7185)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.GraphTerm(SyntaxTreeBuilder.java:3889)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.VarOrTermOrTRefP(SyntaxTreeBuilder.java:8710)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.GraphNodePath(SyntaxTreeBuilder.java:3782)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.ObjectPath(SyntaxTreeBuilder.java:3463)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.ObjectListPath(SyntaxTreeBuilder.java:3040)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.PropertyListPath(SyntaxTreeBuilder.java:2988)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.TriplesSameSubjectPath(SyntaxTreeBuilder.java:2915)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.TriplesBlock(SyntaxTreeBuilder.java:2317)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.BasicGraphPattern(SyntaxTreeBuilder.java:2093)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.GraphPattern(SyntaxTreeBuilder.java:2030)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.GroupGraphPattern(SyntaxTreeBuilder.java:1965)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.WhereClause(SyntaxTreeBuilder.java:1009)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.SelectQuery(SyntaxTreeBuilder.java:373)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.Query(SyntaxTreeBuilder.java:324)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.QueryContainer(SyntaxTreeBuilder.java:212)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.parseQuery(SyntaxTreeBuilder.java:32)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.Bigdata2ASTSPARQLParser.parseQuery2(Bigdata2ASTSPARQLParser.java:336)
entity-resolver_1         | 	... 7 more
entity-resolver_1         | 2018/11/05 12:28:33 Query: SPARQL request failed: 400 Bad Request. Response body: 
entity-resolver_1         | SPARQL-QUERY: queryStr=SELECT ?id
entity-resolver_1         | 			WHERE {
entity-resolver_1         | 				?id a <http://xmlns.com/foaf/0.1/Organization> .
entity-resolver_1         | 				?id <http://www.w3.org/2004/02/skos/core#prefLabel> "National Centre of Scientific Research "Demokritos"" .
entity-resolver_1         | 			}
entity-resolver_1         | java.util.concurrent.ExecutionException: org.openrdf.query.MalformedQueryException: Lexical error at line 4, column 108.  Encountered: "\"" (34), after : "Demokritos"
entity-resolver_1         | 	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
entity-resolver_1         | 	at java.util.concurrent.FutureTask.get(FutureTask.java:192)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.BigdataServlet.submitApiTask(BigdataServlet.java:281)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.QueryServlet.doSparqlQuery(QueryServlet.java:653)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.QueryServlet.doPost(QueryServlet.java:273)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.RESTServlet.doPost(RESTServlet.java:269)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.MultiTenancyServlet.doPost(MultiTenancyServlet.java:193)
entity-resolver_1         | 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
entity-resolver_1         | 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
entity-resolver_1         | 	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:769)
entity-resolver_1         | 	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:585)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
entity-resolver_1         | 	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:577)
entity-resolver_1         | 	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:223)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1125)
entity-resolver_1         | 	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:515)
entity-resolver_1         | 	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:185)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1059)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:215)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:110)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)
entity-resolver_1         | 	at org.eclipse.jetty.server.Server.handle(Server.java:497)
entity-resolver_1         | 	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:311)
entity-resolver_1         | 	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:248)
entity-resolver_1         | 	at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:540)
entity-resolver_1         | 	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:610)
entity-resolver_1         | 	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:539)
entity-resolver_1         | 	at java.lang.Thread.run(Thread.java:748)
entity-resolver_1         | Caused by: org.openrdf.query.MalformedQueryException: Lexical error at line 4, column 108.  Encountered: "\"" (34), after : "Demokritos"
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.Bigdata2ASTSPARQLParser.parseQuery2(Bigdata2ASTSPARQLParser.java:404)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.QueryServlet$SparqlQueryTask.call(QueryServlet.java:716)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.QueryServlet$SparqlQueryTask.call(QueryServlet.java:670)
entity-resolver_1         | 	at com.bigdata.rdf.task.ApiTaskForIndexManager.call(ApiTaskForIndexManager.java:68)
entity-resolver_1         | 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
entity-resolver_1         | 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
entity-resolver_1         | 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
entity-resolver_1         | 	... 1 more
entity-resolver_1         | Caused by: com.bigdata.rdf.sail.sparql.ast.TokenMgrError: Lexical error at line 4, column 108.  Encountered: "\"" (34), after : "Demokritos"
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilderTokenManager.getNextToken(SyntaxTreeBuilderTokenManager.java:3994)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.jj_ntk(SyntaxTreeBuilder.java:9633)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.RDFLiteral(SyntaxTreeBuilder.java:7185)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.GraphTerm(SyntaxTreeBuilder.java:3889)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.VarOrTermOrTRefP(SyntaxTreeBuilder.java:8710)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.GraphNodePath(SyntaxTreeBuilder.java:3782)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.ObjectPath(SyntaxTreeBuilder.java:3463)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.ObjectListPath(SyntaxTreeBuilder.java:3040)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.PropertyListPath(SyntaxTreeBuilder.java:2988)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.TriplesSameSubjectPath(SyntaxTreeBuilder.java:2915)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.TriplesBlock(SyntaxTreeBuilder.java:2317)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.BasicGraphPattern(SyntaxTreeBuilder.java:2093)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.GraphPattern(SyntaxTreeBuilder.java:2030)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.GroupGraphPattern(SyntaxTreeBuilder.java:1965)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.WhereClause(SyntaxTreeBuilder.java:1009)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.SelectQuery(SyntaxTreeBuilder.java:373)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.Query(SyntaxTreeBuilder.java:324)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.QueryContainer(SyntaxTreeBuilder.java:212)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.parseQuery(SyntaxTreeBuilder.java:32)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.Bigdata2ASTSPARQLParser.parseQuery2(Bigdata2ASTSPARQLParser.java:336)
entity-resolver_1         | 	... 7 more
entity-resolver_1         | 2018/11/05 12:28:33 http: panic serving 172.20.0.1:55484: Query: SPARQL request failed: 400 Bad Request. Response body: 
entity-resolver_1         | SPARQL-QUERY: queryStr=SELECT ?id
entity-resolver_1         | 			WHERE {
entity-resolver_1         | 				?id a <http://xmlns.com/foaf/0.1/Organization> .
entity-resolver_1         | 				?id <http://www.w3.org/2004/02/skos/core#prefLabel> "National Centre of Scientific Research "Demokritos"" .
entity-resolver_1         | 			}
entity-resolver_1         | java.util.concurrent.ExecutionException: org.openrdf.query.MalformedQueryException: Lexical error at line 4, column 108.  Encountered: "\"" (34), after : "Demokritos"
entity-resolver_1         | 	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
entity-resolver_1         | 	at java.util.concurrent.FutureTask.get(FutureTask.java:192)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.BigdataServlet.submitApiTask(BigdataServlet.java:281)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.QueryServlet.doSparqlQuery(QueryServlet.java:653)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.QueryServlet.doPost(QueryServlet.java:273)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.RESTServlet.doPost(RESTServlet.java:269)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.MultiTenancyServlet.doPost(MultiTenancyServlet.java:193)
entity-resolver_1         | 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
entity-resolver_1         | 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
entity-resolver_1         | 	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:769)
entity-resolver_1         | 	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:585)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
entity-resolver_1         | 	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:577)
entity-resolver_1         | 	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:223)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1125)
entity-resolver_1         | 	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:515)
entity-resolver_1         | 	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:185)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1059)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:215)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:110)
entity-resolver_1         | 	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)
entity-resolver_1         | 	at org.eclipse.jetty.server.Server.handle(Server.java:497)
entity-resolver_1         | 	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:311)
entity-resolver_1         | 	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:248)
entity-resolver_1         | 	at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:540)
entity-resolver_1         | 	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:610)
entity-resolver_1         | 	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:539)
entity-resolver_1         | 	at java.lang.Thread.run(Thread.java:748)
entity-resolver_1         | Caused by: org.openrdf.query.MalformedQueryException: Lexical error at line 4, column 108.  Encountered: "\"" (34), after : "Demokritos"
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.Bigdata2ASTSPARQLParser.parseQuery2(Bigdata2ASTSPARQLParser.java:404)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.QueryServlet$SparqlQueryTask.call(QueryServlet.java:716)
entity-resolver_1         | 	at com.bigdata.rdf.sail.webapp.QueryServlet$SparqlQueryTask.call(QueryServlet.java:670)
entity-resolver_1         | 	at com.bigdata.rdf.task.ApiTaskForIndexManager.call(ApiTaskForIndexManager.java:68)
entity-resolver_1         | 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
entity-resolver_1         | 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
entity-resolver_1         | 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
entity-resolver_1         | 	... 1 more
entity-resolver_1         | Caused by: com.bigdata.rdf.sail.sparql.ast.TokenMgrError: Lexical error at line 4, column 108.  Encountered: "\"" (34), after : "Demokritos"
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilderTokenManager.getNextToken(SyntaxTreeBuilderTokenManager.java:3994)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.jj_ntk(SyntaxTreeBuilder.java:9633)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.RDFLiteral(SyntaxTreeBuilder.java:7185)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.GraphTerm(SyntaxTreeBuilder.java:3889)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.VarOrTermOrTRefP(SyntaxTreeBuilder.java:8710)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.GraphNodePath(SyntaxTreeBuilder.java:3782)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.ObjectPath(SyntaxTreeBuilder.java:3463)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.ObjectListPath(SyntaxTreeBuilder.java:3040)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.PropertyListPath(SyntaxTreeBuilder.java:2988)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.TriplesSameSubjectPath(SyntaxTreeBuilder.java:2915)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.TriplesBlock(SyntaxTreeBuilder.java:2317)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.BasicGraphPattern(SyntaxTreeBuilder.java:2093)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.GraphPattern(SyntaxTreeBuilder.java:2030)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.GroupGraphPattern(SyntaxTreeBuilder.java:1965)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.WhereClause(SyntaxTreeBuilder.java:1009)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.SelectQuery(SyntaxTreeBuilder.java:373)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.Query(SyntaxTreeBuilder.java:324)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.QueryContainer(SyntaxTreeBuilder.java:212)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.ast.SyntaxTreeBuilder.parseQuery(SyntaxTreeBuilder.java:32)
entity-resolver_1         | 	at com.bigdata.rdf.sail.sparql.Bigdata2ASTSPARQLParser.parseQuery2(Bigdata2ASTSPARQLParser.java:336)
entity-resolver_1         | 	... 7 more
entity-resolver_1         | 
entity-resolver_1         | goroutine 75 [running]:
entity-resolver_1         | net/http.(*conn).serve.func1(0xc0000b60a0)
entity-resolver_1         | 	/usr/local/go/src/net/http/server.go:1746 +0xd0
entity-resolver_1         | panic(0x8b4800, 0xc0003407b0)
entity-resolver_1         | 	/usr/local/go/src/runtime/panic.go:513 +0x1b9
entity-resolver_1         | github.com/sul-dlss-labs/rialto-entity-resolver/handlers.(*findOrganization).Handle(0xc00000cac8, 0xc00043c400, 0x0, 0xc0004e8280, 0x33, 0x8909e0, 0x9d5260, 0x0, 0x9dcbe0)
entity-resolver_1         | 	/go/src/github.com/sul-dlss-labs/rialto-entity-resolver/handlers/find_organization.go:30 +0x2e7
entity-resolver_1         | github.com/sul-dlss-labs/rialto-entity-resolver/generated/restapi/operations.(*FindOrganization).ServeHTTP(0xc0003e4440, 0x9dc620, 0xc00045a000, 0xc00043c400)
entity-resolver_1         | 	/go/src/github.com/sul-dlss-labs/rialto-entity-resolver/generated/restapi/operations/find_organization.go:67 +0x2a7
entity-resolver_1         | github.com/sul-dlss-labs/rialto-entity-resolver/vendor/github.com/go-openapi/runtime/middleware.NewOperationExecutor.func1(0x9dc620, 0xc00045a000, 0xc00043c300)
entity-resolver_1         | 	/go/src/github.com/sul-dlss-labs/rialto-entity-resolver/vendor/github.com/go-openapi/runtime/middleware/operation.go:28 +0x75
entity-resolver_1         | net/http.HandlerFunc.ServeHTTP(0xc00049e180, 0x9dc620, 0xc00045a000, 0xc00043c300)
entity-resolver_1         | 	/usr/local/go/src/net/http/server.go:1964 +0x44
entity-resolver_1         | github.com/sul-dlss-labs/rialto-entity-resolver/vendor/github.com/go-openapi/runtime/middleware.NewRouter.func1(0x9dc620, 0xc00045a000, 0xc00044e300)
entity-resolver_1         | 	/go/src/github.com/sul-dlss-labs/rialto-entity-resolver/vendor/github.com/go-openapi/runtime/middleware/router.go:76 +0x358
entity-resolver_1         | net/http.HandlerFunc.ServeHTTP(0xc0003e5f60, 0x9dc620, 0xc00045a000, 0xc00044e300)
entity-resolver_1         | 	/usr/local/go/src/net/http/server.go:1964 +0x44
entity-resolver_1         | github.com/sul-dlss-labs/rialto-entity-resolver/vendor/github.com/go-openapi/runtime/middleware.Redoc.func1(0x9dc620, 0xc00045a000, 0xc00044e300)
entity-resolver_1         | 	/go/src/github.com/sul-dlss-labs/rialto-entity-resolver/vendor/github.com/go-openapi/runtime/middleware/redoc.go:72 +0x94
entity-resolver_1         | net/http.HandlerFunc.ServeHTTP(0xc0003a0fc0, 0x9dc620, 0xc00045a000, 0xc00044e300)
entity-resolver_1         | 	/usr/local/go/src/net/http/server.go:1964 +0x44
entity-resolver_1         | github.com/sul-dlss-labs/rialto-entity-resolver/vendor/github.com/go-openapi/runtime/middleware.Spec.func1(0x9dc620, 0xc00045a000, 0xc00044e300)
entity-resolver_1         | 	/go/src/github.com/sul-dlss-labs/rialto-entity-resolver/vendor/github.com/go-openapi/runtime/middleware/spec.go:46 +0x73
entity-resolver_1         | net/http.HandlerFunc.ServeHTTP(0xc0003a1000, 0x9dc620, 0xc00045a000, 0xc00044e300)
entity-resolver_1         | 	/usr/local/go/src/net/http/server.go:1964 +0x44
entity-resolver_1         | net/http.serverHandler.ServeHTTP(0xc0004204e0, 0x9dc620, 0xc00045a000, 0xc00044e300)
entity-resolver_1         | 	/usr/local/go/src/net/http/server.go:2741 +0xab
entity-resolver_1         | net/http.(*conn).serve(0xc0000b60a0, 0x9dcb20, 0xc00023c380)
entity-resolver_1         | 	/usr/local/go/src/net/http/server.go:1847 +0x646
entity-resolver_1         | created by net/http.(*Server).Serve
entity-resolver_1         | 	/usr/local/go/src/net/http/server.go:2851 +0x2f5
```

--- a/repository/sparql_reader.go
+++ b/repository/sparql_reader.go
@@ -3,6 +3,7 @@ package repository
 import (
 	"fmt"
 	"log"
+	"strings"
 	"time"
 
 	"github.com/knakk/sparql"
@@ -42,7 +43,7 @@ func (r *SparqlReader) QueryByEntityTypeIdentifierTypeAndObject(entityType strin
 			WHERE {
 				?id a <%s> .
 				?id <http://purl.org/dc/terms/identifier> "%s"^^<%s> .
-			}`, entityType, object, identifierType)
+			}`, entityType, r.escapeLiteral(object), identifierType)
 	results, err := r.repo.Query(query)
 	if err != nil {
 		log.Printf("[SPARQL] %s returned error: %v", query, err)
@@ -65,7 +66,7 @@ func (r *SparqlReader) QueryByTypePredicateAndObject(entityType string, predicat
 			WHERE {
 				?id a <%s> .
 				?id <%s> "%s" .
-			}`, entityType, predicate, object)
+			}`, entityType, predicate, r.escapeLiteral(object))
 	results, err := r.repo.Query(query)
 	if err != nil {
 		log.Printf("[SPARQL] %s returned error: %v", query, err)
@@ -80,4 +81,8 @@ func (r *SparqlReader) QueryByTypePredicateAndObject(entityType string, predicat
 	id := results.Solutions()[0]["id"].String()
 	log.Printf("[SPARQL] %s returned %s from %d results", query, id, len(results.Solutions()))
 	return &id, nil
+}
+
+func (r *SparqlReader) escapeLiteral(literal string) string {
+	return strings.Replace(literal, "\"", "\\\"", -1)
 }

--- a/repository/sparql_reader_test.go
+++ b/repository/sparql_reader_test.go
@@ -64,3 +64,26 @@ func TestQueryByTypePredicateAndObject(t *testing.T) {
 	result, _ := reader.QueryByTypePredicateAndObject("http://xmlns.com/foaf/0.1/Person", "http://www.w3.org/2006/vcard/ns#fn", "Giarlo, Mike")
 	assert.Equal(t, *result, "http://sul.stanford.edu/rialto/agents/people/8de0ce5e-a2a4-4e61-974e-df6c213cf220")
 }
+
+func TestQueryByTypePredicateAndObjectEscaping(t *testing.T) {
+	fakeRepo := new(MockedRepo)
+
+	institutionJSON := strings.NewReader(`{
+	    "head": { "vars": [ "id" ] } ,
+	    "results": {
+	      "bindings": [
+	        {
+	          "id": { "type": "uri" , "value": "http://sul.stanford.edu/rialto/agents/people/8de0ce5e-a2a4-4e61-974e-df6c213cf220" }
+	        }
+	      ]
+	    }
+	  }`)
+	fakeRepo.On("Query", "SELECT ?id\n\t\t\tWHERE {\n\t\t\t\t?id a <http://xmlns.com/foaf/0.1/Person> .\n\t\t\t\t?id <http://www.w3.org/2006/vcard/ns#fn> \"Giarlo, Mike \\\"Mikey\\\"\" .\n\t\t\t}").
+		Return(sparql.ParseJSON(institutionJSON))
+
+	reader := &SparqlReader{
+		repo: fakeRepo,
+	}
+	result, _ := reader.QueryByTypePredicateAndObject("http://xmlns.com/foaf/0.1/Person", "http://www.w3.org/2006/vcard/ns#fn", "Giarlo, Mike \"Mikey\"")
+	assert.Equal(t, *result, "http://sul.stanford.edu/rialto/agents/people/8de0ce5e-a2a4-4e61-974e-df6c213cf220")
+}

